src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -149,6 +149,77 @@ export async function getSistrResults(submissionId) {
   }
 }
 
+/**
+ * Get the output file info
+ * @param {number} submissionID Submission ID
+ * @return {Promise<*>} `data` contains the OK response; `error` contains error information if an error occurred.
+ */
+export async function getOutputInfo(submissionId) {
+  try {
+    const res = await axios.get(`${URL}/${submissionId}/outputs`);
+    return res.data;
+  } catch (error) {
+    return { error };
+  }
+}
+
+/**
+ * Get the data from the output file for with the supplied chunk size
+ * @param {object} contains the output file data
+ * @return {Promise<*>} `data` contains the OK response; `error` contains error information if an error occurred.
+ */
+export async function getDataViaChunks(outputFileInfoObject) {
+  try {
+    const res = await axios.get(
+      `${URL}/${outputFileInfoObject.submissionId}/outputs/${outputFileInfoObject.fileId}`,
+      {
+        params: {
+          seek: outputFileInfoObject.seek,
+          chunk: outputFileInfoObject.chunk
+        }
+      }
+    );
+    return res.data;
+  } catch (error) {
+    return { error };
+  }
+}
+
+/**
+ * Get the file output from line x upto line y.
+ * @param {object} contains the output file data
+ * @return {Promise<*>} `data` contains the OK response; `error` contains error information if an error occurred.
+ */
+export async function getDataViaLines(outputFileInfoObject) {
```

Comment:
Same here with param destructuring

Reply:
Updated in d57854982494fb9733f7243e6d74f315e1ee9297